<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculsTransits</class>
 <widget class="QFrame" name="CalculsTransits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QCheckBox" name="calcTransitLunaireJour">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>136</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Inclure transits/conjonctions lunaires de jour</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_52">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>108</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_transit2">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_elongationMaxCorps">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Élongation maximale avec le corps :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="elongationMaxCorps">
      <property name="maximumSize">
       <size>
        <width>51</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calculsTransit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>188</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Calculs</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>82</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_transit1">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout_transit1">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_hauteurSatTransit">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Hauteur minimale du satellite :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="hauteurSatTransit">
        <property name="font">
         <font/>
        </property>
        <property name="currentText">
         <string notr="true">10°</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>0°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autre...</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_7">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="valHauteurSatTransit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>1</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="inputMask">
        <string notr="true">99</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>6</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_dateTransits">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_dateInitialeTransit">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date initiale :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_dateFinaleTransit">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date finale :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateFinaleTransit">
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateInitialeTransit">
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="effacerHeuresTransit">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Effacer heures</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="majElementsOrbitauxIss">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>188</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre à jour Éléments orbitaux de l'ISS...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxChoixCorps">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Corps</string>
   </property>
   <widget class="QCheckBox" name="soleilTransit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Soleil</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="luneTransit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>44</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Lune</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_51">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_lieuTransit">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_lieuxObservation">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Lieu d'observation :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="lieuxObservation">
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="currentText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="parametrageDefautTransit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>188</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Paramétrage par défaut</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listeTransits">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>51</y>
     <width>210</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="filtreSatellites">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>27</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true" extracomment="Filter satellite names"/>
   </property>
   <property name="placeholderText">
    <string>Filtre</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>278</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_ageElem">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="ageElementsOrbitauxTransit1">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string notr="true">-360.3 </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_ageElementsOrbitauxTransit">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Ages des éléments orbitaux : [</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="ageElementsOrbitauxTransit2">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string notr="true">-360.3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lbl_jours">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>] jours</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="majElementsOrbitaux">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>158</y>
     <width>146</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre à jour...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="satellitesChoisis">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
